tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/entity/TokenImpl.java

```diff
@@ -0,0 +1,52 @@
+package com.google.cloud.bigquery.utils.queryfixer.entity;
+
+import org.apache.calcite.sql.parser.babel.Token;
+
+/**
+ * An implementation of token interface based on the Token class of Babel Calcite Parser.
+ * */
+public class TokenImpl implements IToken {
+
+  private final Token token;
+
+  public TokenImpl (Token token) {
+    this.token = token;
+  }
+
+  @Override public int getKind() {
+    return token.kind;
+  }
```

Comment:
nit: not sure this has been auto formatted yet... i think usually this should be
@Override
public int getKind() {
...
}